pkg/model/metric.go

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) 2017 Huawei Technologies Co., Ltd. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+This module implements the common data structure.
+
+*/
+
+package model
+
+// VolumeSpec is an block device created by storage service, it can be attached
+// to physical machine or virtual machine instance.
+type CollectMetricSpec struct {
```

Comment:
I think `CollectMetricSpec` and `GetMetricSpec` should be removed here, because we only define resource schema here.

Reply:
the comments were incorrect, updated.